runbotics/runbotics-orchestrator-ui/src/src-app/views/credentials/CredentialsCollection/EditCredentialsCollection/SharedWithUsers/SharedWithUsers.tsx

```diff
@@ -32,42 +31,44 @@ export const SharedWithUsers: FC<SharedWithUsersProps> = ({ credentialsCollectio
 
     const { user: collectionCreator } = useAuth();
     const [selectedUsers, setSelectedUsers] = useState(credentialsCollectionFormState.sharedWith || []);
-    const [filteredUsers, setFilteredUsers] = useState([]);
+    const [availableUsers, setAvailableUsers] = useState(filteredSharableUsers('', nonAdmins.all, { sharedWithUsers: credentialsCollectionFormState.sharedWith, selectedUsers, collectionCreatorId: collectionCreator.id}));
 
     useEffect(() => {
-        setFilteredUsers(filteredSharableUsers('', nonAdmins.all, { sharedWithUsers: credentialsCollectionFormState.sharedWith, selectedUsers, collectionCreatorId: collectionCreator.id}));
+        setAvailableUsers(filteredSharableUsers('', nonAdmins.all, { sharedWithUsers: credentialsCollectionFormState.sharedWith, selectedUsers, collectionCreatorId: collectionCreator.id}));
     }, [nonAdmins.all]);
 
     useEffect(() => {
         dispatch(usersActions.getActiveNonAdmins());
     }, []);
 
     useEffect(() => {
-        setCredentialsCollectionFormState((prevFormState) => ({
+        setCredentialsCollectionFormState(prevFormState => ({
             ...prevFormState,
             accessType: selectedUsers.length > 0 ? AccessType.GROUP : AccessType.PRIVATE,
-            sharedWith: selectedUsers,
+            sharedWith: selectedUsers
```

Comment:
we've decided to keep ',' at the end of the last property in objects because when someone add another line there's git blame for only one, not two lines